templates/disbursements.html

```diff
@@ -17,22 +17,23 @@
   </div>
 </section>
 
-<div id="datatable-modal" class="modal__overlay" aria-hidden="true">
-  <div class="modal">
-    <div class="modal__row">
-      <button title="Close menu" class="js-hide modal__close" data-hides="datatable-modal"><i class="ti-close"></i></button>
+<div id="datatable-modal" class="panel__overlay" aria-hidden="true">
```

Comment:
We should probably make this a partial as it's quite a bit of duplicated code. Although, does it only appear in these two files, or is there a chance it will be in more files?